asc2log.c

```diff
@@ -264,6 +272,10 @@ void eval_canfd(char* buf, struct timeval *date_tvp, char timestamps, int dplace
 	if (strlen(dir) != 2) /* "Rx" or "Tx" */
 		return;
 
+	/* check for signed integer overflow */
+	if (dplace == 4 && read_tv.tv_usec >= INT_MAX/100)
+		return;
+
```

Comment:
No check for dplace == 5 as in eval_can() ?

Reply:
Yes seems that I forgot this.

Reply:
Should be there now